src/systems/filecoin_files/data_transfer/_index.md

```diff
@@ -3,4 +3,92 @@ menuTitle: Data Transfer
 title: "Data Transfer in Filecoin"
 ---
 
+_Data Transfer_ is a system for transferring all or part of a `Piece` across the network when a deal is made.
```

Comment:
curious if you have opinions about the name @hannahhoward -- "Data Transfer" sounds somewhat bland and potentially more generic/less useful than it should be. Wonder if some name closer to Graphsync may be worth rebinding too. (i called it this because i didnt know what to call it at the time yet, and didnt want to introduce a superfluous name)